brainstormforce/ultimate-addons-for-gutenberg
# Fix: Call to undefined function `twentyseventeen_panel_count()` when previewing another theme from the customizer.

### Description

- https://brainstormforce.slack.com/archives/CAQ2TSA6A/p1633516341100000

### Types of changes




- Bug fix ( breaking change which fixes an issue)

### How has this been tested?

- Go to customizer > click on Change > Change theme to Astra or any
- Check Live preview of any theme

### Checklist:
